nrf_security/src/mbedtls/CMakeLists.txt

```diff
@@ -135,6 +135,7 @@ if (CONFIG_ENTROPY_CC310)
   zephyr_library_sources(${NRF_SECURITY_ROOT}/src/backend/cc310/replacements/entropy.c)
 elseif(CONFIG_ENTROPY_NRF5_RNG)
   zephyr_library_sources(${NRF_SECURITY_ROOT}/src/backend/nrf5x/entropy_nrf5x.c)
+  zephyr_library_sources(${ARM_MBEDTLS_PATH}/library/entropy.c)
```

Comment:
> -Adding building of entropy.c as OpenThread require to use the APIs

can you elaborate ?
Why is it only required in OpenThread case?

Reply:
They utilize a mechanism for getting entropy that is different from the rest of Zephyr (as they interface more directly to mbed TLS). They need access to the APIs directly, and these APIs haven't been made available otherwise